addons/google_calendar/models/google_calendar.py

```diff
@@ -546,14 +538,12 @@ def synchronize_events_cron(self, cr, uid, context=None):
                 _logger.info("[%s] Calendar Synchro - Exception : %s !" % (user_to_sync, exception_to_unicode(e)))
         _logger.info("Calendar Synchro - Ended by cron")
 
-    def synchronize_events(self, cr, uid, ids, lastSync=True, context=None):
-        if context is None:
-            context = {}
+    def synchronize_events(self, lastSync=True):
```

Comment:
@msh-odoo  :+1: and @rgo-odoo  if you not give decorator then take default as that is ***@api.multi***  see in [Guideline](http://odoo-new-api-guide-line.readthedocs.org/en/latest/decorator.html) Under 3rd point mention that things